build.gradle

```diff
@@ -58,12 +60,14 @@ allprojects {
 
 }
 
+
+
 configure(javaProjects) { subproject ->
 	apply plugin: 'java-library'
 	apply plugin: 'eclipse'
 	apply plugin: 'idea'
-
 	apply from: "${rootDir}/publish-maven.gradle"
+	apply from: "${rootDir}/gradle/checkstyle-conventions.gradle"
```

Comment:
Since our `checkstyle-conventions.gradle` is already short enough I don't see a reason in the separate file.
I also wonder why it still fails even if we have that `ignoreFailures = true`...